docs/howto/openstack/nova/rescue-server.md

```diff
@@ -0,0 +1,89 @@
+# Rescue a server
+
+This guide will walk you through the required steps to access the
+the rescue mode when you need to.
+
+1. Access the server operating system disk to fix corrupted file system.
+2. Reset access credential, login password.
+
+## Prerequisites
+
+You need to have a server you wish to rescue. Additionally, if you
+prefer to work with the OpenStack CLI, then make sure to properly
+[enable it first](/howto/getting-started/enable-openstack-cli/).
+
+[Rescue](https://docs.openstack.org/nova/latest/user/rescue.html)
+(or Server rescue) is the ability to boot the server on additional cloud image
+like system-rescue that shipped with variety tools to fix Operating System disk
+file system or reset login password or the other option is using the base
+server image, rescue server using base image support only the server boot from
+Ephemeral disk and did not support the server boot from volume,
+rescue operation is a two-steps process for the user:
+
+1. Initiate the rescue.
+2. Access the server either using CCP2 >_ Remote Console or connect to server
+   using ssh server floating IP
+
+
+## Initiating the rescue
+
+=== "{{gui}}"
+    Navigate to the server list.
+
+    ![The left hand side navigation panel, with the word "Servers" highlighted.](assets/rescue-server/01-left-side-panel.png)
+
+    Find the server you want to rescue in the list, and on the
+    right-hand side click on its menu button.
+
+    ![An orange circle with three white dots.](assets/rescue-server/02-menu-button.png)
+
+    Click on _Rescue Server_.
+
+    ![A list of server actions, with the line "Rescue Server" highlighted.](assets/rescue-server/03-menu-list.png)
+
+    Once selected, the rescue dialog appear with details included for region
+    name and the server details
+
+    Click _No_ to cancel or _Yes, Rescue_ to proceed
+
+    ![About to rescue a server, the effected details about the Server name, UUID and region with a question to confirm the rescue on red button as "Yes, Rescue" or "No" to cancel.](assets/rescue-server/04-rescue-button.png)
+
+    Server on Rescue mode
+
+    ![exclamation mark on server icon showing the server on "rescue mode".](assets/rescue-server/05-rescue-mode.png)
+
+=== "OpenStack CLI"
+    To start the rescue using system-rescue image use the following command:
+
+    ```bash
+    for uuid in `openstack image list --tag system-rescue -c ID -f value`; do openstack --os-compute-api-version 2.87 server rescue --image $uuid <server_id> ; done
+    ```
+
+    To start the rescue using server base image use the following command:
+
+    ```bash
+    openstack server rescue <server_id>
```

Comment:
I think this method should fail for BFV instances. We should be explicit on when to use rescue image. Given that these docs are not actually considering non-BFV, not sure we should even mention this (unless I'm wrong and bfv instances can boot with base image)